### PR TITLE
solve the task

### PR DESCRIPTION
**Pull request by Cerne13**

### CODE REVIEW

**Review by Vasyl-Poremchuk — APPROVED** [review]
GJ!

File: app/main.py [hunk 1]
```diff
@@ -1,5 +1,70 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int):
```
Comment (Nattalli) [inline app/main.py:2]: add type annotation for function. if it returns nothing then use `-> None`

File: app/main.py [hunk 2]
```diff
@@ -1,5 +1,70 @@
...
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords:
+            list[int] = None
```
Comment (Nattalli) [inline app/main.py:12]: can it be in one line?

File: app/main.py [hunk 3]
```diff
@@ -1,5 +1,70 @@
...
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords:
+            list[int] = None
```
Comment (Nattalli) [inline app/main.py:40]: and here

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Cerne13) [reply app/main.py:2]: Thank you!

**Review by Nattalli — APPROVED** [review]
Good job! 😸

**Pull request closed without merging**